docs/main/faqs/faqs.md

```diff
@@ -664,6 +664,12 @@ It is recommended that Replicated PV Mayastor volumes be provisioned with a mini
 
 [Go to top](#top)
 
+### Should I install the Replicated PV Mayastor/Mayastor Helm chart alongside the OpenEBS/OpenEBS Helm release?
+
+The Replicated PV Mayastor/Mayastor Helm chart, along with other stable OpenEBS storage plugins, is included as a dependency within the OpenEBS/OpenEBS Helm chart. The OpenEBS/OpenEBS chart can be configured to deploy the Replicated PV Mayastor storage engine. It is not recommended to deploy the Helm chart dependencies independently alongside the umbrella chart (OpenEBS/OpenEBS). This may lead to configuration conflicts and operational issues.
```

Comment:
```suggestion
The mayastor/mayastor Helm chart, along with other stable OpenEBS storage plugins, is included as a dependency within the openebs/openebs Helm chart. The openebs/openebs chart can be configured to deploy the Replicated PV Mayastor storage engine. It is not recommended that you deploy the Helm chart dependencies independently alongside the umbrella chart (openebs/openebs). This may lead to configuration conflicts and operational issues.
```